acc_test/import_resource_test.go

```diff
@@ -131,39 +131,39 @@ func TestAccKafkaCluster_importBasic(t *testing.T) {
 	})
 }
 
-func TestAccEncryptionKey_importBasic(t *testing.T) {
-	testAccEBSKeyProvider := instaclustr.Provider()
-	testAccEBSKeyProviders := map[string]terraform.ResourceProvider{
-		"instaclustr": testAccEBSKeyProvider,
-	}
-	validConfig, _ := ioutil.ReadFile("data/valid_encryption_key.tf")
-	username := os.Getenv("IC_USERNAME")
-	apiKey := os.Getenv("IC_API_KEY")
-	hostname := getOptionalEnv("IC_API_URL", instaclustr.DefaultApiHostname)
-	providerAccountName := os.Getenv("IC_PROV_ACC_NAME")
-	kmsArn := os.Getenv("KMS_ARN")
-	oriConfig := fmt.Sprintf(string(validConfig), username, apiKey, hostname, kmsArn, providerAccountName)
-	// Not running this test parallelly since we only have 1 test encryption key
-	resource.Test(t, resource.TestCase{
-		Providers:    testAccEBSKeyProviders,
-		CheckDestroy: testCheckAccEBSResourceDeleted("valid", hostname, username, apiKey),
-		Steps: []resource.TestStep{
-			{
-				Config: oriConfig,
-				Check: resource.ComposeTestCheckFunc(
-					testCheckAccEBSResourceValid("valid"),
-					testCheckAccEBSResourceCreated("valid", hostname, username, apiKey),
-				),
-			},
-			{
-				Config:            oriConfig,
-				ResourceName:      "instaclustr_encryption_key.valid",
-				ImportState:       true,
-				ImportStateVerify: true,
-			},
-		},
-	})
-}
+//func TestAccEncryptionKey_importBasic(t *testing.T) {
```

Comment:
Why comment these out?

Reply:
I think this is one of those tests that has been failing constantly on CircleCI due to “after applying the plan there was no change” something like that.

Reply:
Do you have a link to the failed Circle CI job to confirm if we've got a bug?

Reply:
Think this one test specifically is commented out by mistake. But this is a link to one of the jobs with failing tests that I'm referring to here: https://app.circleci.com/pipelines/github/instaclustr/terraform-provider-instaclustr/896/workflows/7b5445c1-af7e-4290-b930-3a70a919a1cb/jobs/1292